tests/unittest/_torch/multi_gpu/test_mnnvl_allreduce.py

```
@@ -202,6 +207,55 @@ def run_reject_single_rank(tensor_parallel_size, single_rank_forward_func,
     return True
 
 
+@torch.inference_mode()
+def run_mnnvl_rms_norm_single_rank(
+    tensor_parallel_size: int,
+    input: torch.Tensor,
```

Ruff (0.16.2)

[error] Function argument `input` is shadowing a Python builtin

(A002)